v4/build.gradle

```diff
@@ -21,11 +21,11 @@ ext {
   // Make interlokServiceTester an explicit version
   // If interlokVersion == SNAPSHOT then server-tester follows snapshot, otherwise it should
   // be pinned to it's own release.
-  interlokVersion = project.findProperty('interlokVersion') ?: '4.2.0-RELEASE'
-  interlokServiceTesterVersion = project.findProperty('interlokServiceTesterVersion') ?: '4.2.1-RELEASE'
+  interlokVersion = project.findProperty('interlokVersion') ?: '4.3.0-RELEASE'
+  interlokServiceTesterVersion = project.findProperty('interlokServiceTesterVersion') ?: '4.3.0-RELEASE'
   interlokUiVersion = project.findProperty('interlokUiVersion') ?: interlokVersion
   verifyReportConfigCheckAvailable = '3.12-SNAPSHOT'
-  requiresServiceTesterPatch = "4.2.0-RELEASE"
+  requiresServiceTesterPatch = "4.3.0-RELEASE"
```

Comment:
Please don't blindly change version numbers...
The only version number that needs to change is `interlokVersion` -> READ the comments !
```
if interlokVersion = 4.2.0-RELEASE -> we need to use serviceTesterVersion 4.2.1-RELEASE because it was broken
if interlokVersion = 4.1.0-RELEASE -> we need to use serviceTesterVersion 4.1.0-RELEASE
if interlokVersion = 4.3-SNAPSHOT -> we use serviceTester 4.3-SNAPSHOT
if intelrokVersion = 4.3.0-RELEASE -> we use serviceTester 4.3.0-RELEASE
if interlokVersion = 4.4-SNAPSHOT -> we use serviceTester 4.4-SNAPSHOT
```

Reply:
Well, I didn't blindly change the version number... I updated it to the latest release. 
I made the decision to leave the service tester patch part in; just in case it was needed again in the future.
I was going to discuss removing it with seb on Monday. I thought this would be the quickest way forward, to get the parent up to date. I should of made this clear in the PR comments / commit message, sorry about that.

Reply:
Yes, but updating all occurences of 4.2.0-RELEASE to 4.3.0-RELEASE was the wrong thing to do...
_requiresServiceTesterPatch should still stay at 4.2.0-RELEASE_ and interlokServiceTesterVersion should stay at 4.2.1-RELEASE because then the logic that is required is fulfilled.

Then you test the build, to see if having interlokVersion=4.3.0-RELEASE gives you the right thing for interlokServiceTester when you run a version check.

Reply:
Okay, I've updated versions based on your feedback.